Multisig/UI/Settings/AppSettingsViewController/DesktopPairing/DesktopPairingViewController.swift

```diff
@@ -31,6 +35,12 @@ class DesktopPairingViewController: UITableViewController {
         tableView.registerHeaderFooterView(DesktopPairingHeaderView.self)
         tableView.sectionHeaderHeight = UITableView.automaticDimension
 
+        infoButton = UIBarButtonItem(image: UIImage.init(named: "ico-info"),
```

Comment:
This button looks too small. Is there an easy way to resize it?

Reply:
isn't this part of https://app.zenhub.com/workspaces/mobile-engineering-60cc78740f488d000e996335/issues/gnosis/safe-ios/1899?